examples/sms/Receiving/buildozer.spec

```diff
@@ -186,7 +186,7 @@ android.manifest.intent_filters = AndroidManifest.xml
 log_level = 2
 
 # (int) Display warning if buildozer is run as root (0 = False, 1 = True)
-warn_on_root = 1
+warn_on_root = 0
```

Comment:
I need to run `sudo buildozer android debug deploy run` in my machine else I get an error `Permission denied`. Therefore I made it 0 to avoid warning.